src/pubtools/sign/ansible/clear_sign.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
```

Comment:
Can you please rename this msg_clear_sign as this module provides the functionality for specific signer (MsgSigner)

Reply:
Done.